package.json

```diff
@@ -15,7 +15,7 @@
     "build-stage": "NODE_ENV=stage node build/build.js"
   },
   "dependencies": {
-    "common-component": "git+https://github.com/Rise-Vision/common-component.git#v1.8.1",
+    "common-component": "git+https://github.com/Rise-Vision/common-component.git#feature/licensing-for-rpp",
```

Comment:
@jungeunyoon The use of your `common-component/licensing` isn't approved in the PR https://github.com/Rise-Vision/common-component/pull/31 , there are still open questions and requests for changes. I don't think this PR here should be reviewed until that one is approved. Thoughts?

Reply:
@stulees agreed, thanks!

Reply:
@jungeunyoon why don't you create a 1.8.2 release back on common-component, just to have the numeric reference of the required version ?

Reply:
@santiagonoguez Yup, side affect from using branches. Fixed, please review!